sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/SchemaCoder.java

```diff
@@ -119,27 +86,6 @@ protected SchemaCoder(
     return RowCoder.of(schema);
   }
 
-  /** Returns the coder used for a given primitive type. */
-  public static <T> Coder<T> coderForFieldType(FieldType fieldType) {
```

Comment:
I find this was useful to have, and it was public. Instead, I would suggest `SchemaCoderHelpers` to be package-private, and this method to delegate to `SchemaCoderHelpers`.

Reply:
done